src/js/services/conversation-service.js

```diff
@@ -0,0 +1,76 @@
+import { store } from 'stores/app-store';
+import { messageAdded, setConversation } from 'actions/conversation-actions';
+import { setFayeSubscription, unsetFayeSubscription } from 'actions/faye-actions';
+import { core } from 'services/core';
+import { immediateUpdate } from 'services/user-service';
+import { initFaye } from 'utils/faye';
+
+export function sendMessage(text) {
+    var sendFn = () => {
+        const message = {
+            text: text,
+            role: 'appUser'
+        };
+
+        store.dispatch(messageAdded(message));
```

Comment:
Currently nothing, but that was the behavior before. We should definitely do something globally about that sort of thing in the `catch` of these promises. 

If we add it only on server response, it creates a unacceptable delay for the user.